### PR TITLE
add recipe for ptac

### PR DESCRIPTION
**Pull request by serrayos**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by serrayos** [comment]
@conda-forge/help-python ready for review! Thank you in advance

File: recipes/ptac/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+requirements:
+  build:
+    - python >=3.8
+    - setuptools ==63.4.1
+    - wheel ==0.37.1
+
+  host:
+    - python >=3.8
+    - pip
```
Comment (dopplershift) [inline recipes/ptac/meta.yaml:25]: If these are needed, they should all be under `host` and eliminate the `build` section. Also, does it really need specific versions of setuptools and wheel to build the package?

**Comment by dopplershift** [comment]
I also need confirmation from @SimonNieland that they are willing to be listed as a maintainer for this recipe.

**Comment by SimonNieland** [comment]
I confirm to be willing to be listed as a maintainer for the package.

File: recipes/ptac/meta.yaml (hunk 1 above)
Comment (serrayos) [reply recipes/ptac/meta.yaml]: @dopplershift Thanks for the feedback! I just updated the requirements

**Review by dopplershift — APPROVED** [review]

**Comment by serrayos** [comment]
@dopplershift thanks so much! The packages is now only available on noarch , but I would like to make it available also on linux, osx and windows. Which changes are necessary for that?

**Comment by dopplershift** [comment]
A noarch package means that it is installable on ALL platforms. Are you having trouble installing `ptac` in one of those environments?

**Pull request merged**